microsoft-edge/devtools-guide-chromium/reasons-edge-devtools.md

```diff
@@ -0,0 +1,215 @@
+
+![The 3D View tool.](reasons-edge-devtools-images/3d-tool.png)
+
+See [Navigate z-index, DOM, and layers using the 3D View tool](3d-view/index.md).
+
+
+### Integration with webhint
+
+The **Issues** tool automatically shows you all the things that need fixing in the current webpage.  Under the hood, the **Issues** tool uses [webhint](https://webhint.io), which generates reports and provides reporting on various issues.  You not only learn what's wrong with your webpage, but also why it's a problem, where in your webpage the problem exists, and how to fix the issue.
+
+Current problems of a webpage displayed in the Issues panel:
+
+![Current problems of a webpage displayed in the Issues panel.](reasons-edge-devtools-images/webpage-problems-displayed-in-issues-panel.png)
+
+See [Find and fix problems using the Issues tool](issues/index.md).
+
+
+### Integration of Microsoft Edge DevTools with Visual Studio Code
+
+When you're debugging a problem with your webpage, it's cumbersome to change from a code editor, to the browser, then using DevTools in the browser to analyze the problem, and then going back to try a possible fix of the problem in your source code.  This is why we created a Visual Studio Code extension that embeds Microsoft Edge DevTools into your code editor workflow. 
+
+For example, without integrated tools, you temporarily fix CSS style rules in your transient, resulting webpage files that are sent from your development server to your browser, and then you have to remember the changes, and then re-apply those changes to your actual source code in your source files, over in your code editor.
+
+Now, by using the Microsoft Edge DevTools extension for Visual Studio Code, you can simply use Microsoft Edge DevTools to change your source code directly within an instance of DevTools that's running in the IDE environment of Visual Studio Code.<!-- condense end -->
+
+Microsoft Edge DevTools integrated into Visual Studio Code:
+
+![Microsoft Edge DevTools integrated into Visual Studio Code.](reasons-edge-devtools-images/devtools-integrated-into-visual-studio-code.png)
+
+See [Microsoft Edge DevTools extension for Visual Studio Code](../visual-studio-code/microsoft-edge-devtools-extension.md).
+
+
+### Network Console
+
+Inspecting the network traffic of your web product is already insightful, but often you need to make some changes to the requests to see why they fail.  Using the **Network Console**, you can change and replay any of the requests, and you can make detailed API calls.
+
+**Network Console** showing options for changing a network call:
+
+![Network Console showing options for changing a network call.](reasons-edge-devtools-images/network-console-options-changing-network-call.png)
```

Comment:
Why is there a red box around the **Network** tool in this screenshot? The text doesn't mention this tool, so I'm not sure why attention is called out.